assets/js/base/ProjectLoader.js

```diff
@@ -465,7 +465,11 @@ let ProjectLoader = function(container, url, recommended_by_project_id, recommen
       
       case '#mostViewed':
         return  '<div><i class="fas fa-eye program-small-icon"></i>' + project.Views + '</div>'
-      
+  
+      case '#scratchRemixes':
+        return  '<div><i class="fas fa-eye program-small-icon"></i>' + project.Views + '</div>'
```

Comment:
line 470-472 can be removed since this case does the same as 
case '#recommendations':
case '#more-from-this-user-recommendations':

Reply:
Done